plaidml/edsl/edsl_test.cc

```diff
@@ -143,9 +143,9 @@ TEST(CppEdsl, BitLeft) {
   }
 }
 
-TEST(CppEdsl, BitRightSignedScalarSigned) {
+TEST(CppEdsl, BitRightSignedScalar) {
   auto A = Placeholder(DType::INT64, {3, 3});
-  auto C = A >> 1;
+  auto C = A >> std::uint64_t{1};
```

Comment:
I actually think we should let users use integers directly, but then at a lower level prevent non-negative integers from being specified.

Reply:
@jbruestle Do you have any thoughts on this? I was hoping we could have only 2 constant types: `double` and `int64_t`, but really these ought to be arbitrary precision types.